src/app/project/project-documents/project-documents.component.ts

```diff
@@ -925,6 +932,7 @@ export class ProjectDocumentsComponent implements OnInit, OnDestroy {
     );
 
     if (docType === Constants.documentTypes.CATEGORIZED) {
+      console.log(this.filterForAPI);
```

Comment:
Can we use the default logger here instead if we need this, otherwise we should remove the console.log

Reply:
The log is probably unecessary altogether. I'll just strip them out